src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -61,20 +62,23 @@ class MarathonSchedulerActor private (
   val lockedRunSpecs = collection.mutable.Map[PathId, Int]().withDefaultValue(0)
   var historyActor: ActorRef = _
   var activeReconciliation: Option[Future[Status]] = None
+  var stateSubscription: Cancellable = _
+  implicit val materializer = ActorMaterializer()
```

Comment:
What do you think about `ActorMaterializer(ActorMaterializerSettings(context.system))`?

Reply:
ActorMaterializer takes the actor context as the default

Reply:
We discussed; to rehash, benefit of using default is that materializer will enclose the actor context and will be automatically shut down if this actor is shut down.